plotting_tools/PlotHelpers.h

```diff
@@ -0,0 +1,222 @@
+TH1D* get_hist_with_same_width_bins(TH1D* input_hist, string input_hist_name, vector<int> lxy_bins, float rebin, float int_lumi, float cross_sec, float BR, float N_tot)
```

Comment:
Depending on how you call this function maybe you don't need to pass `input_hist_name`, as it can be extracted from the `input_hist` itself (`input_hist->GetName()`).